### DISCUSSION TITLE
Why was 1.6.241106002 unlisted without any notices?

### DISCUSSION THREAD

**Discussion by HotCakeX** [microsoft/WindowsAppSDK#4898] [Q&A]
I received Dependbot alert for version `1.6.241114003` and updated to it in Visual studio too. I came to GitHub to check for release notes to see what's new/changed but there is nothing.

I checked Nuget and the version i was previously using is unlisted.

![image](https://github.com/user-attachments/assets/47af71f7-9b3a-4b32-91a3-92a5537719f9)

Is there really a security vulnerability in the version i was using? if so please let us know so we can update our in production apps that were using that unlisted version. thanks

**Comment by ghost1372** [comment]
Duplicate
https://github.com/microsoft/WindowsAppSDK/issues/4887

**Comment by HotCakeX** [comment]
Dev response: https://github.com/microsoft/WindowsAppSDK/issues/4881#issuecomment-2480939942
